RELEASING.md

```diff
@@ -0,0 +1,29 @@
+# Release Checklist
+
+- [ ] check tests pass on [GitHub Actions](https://github.com/python/cherry-picker/actions)
+      [![GitHub Actions status](https://github.com/python/cherry-picker/actions/workflows/main.yml/badge.svg)](https://github.com/python/cherry-picker/actions/workflows/main.yml)
+
+- [ ] Update [changelog](https://github.com/python/cherry-picker#changelog)
+
+- [ ] Go to https://github.com/python/cherry-picker/releases
+
+- [ ] Click "Draft a new release"
+
+- [ ] Click "Choose a tag"
+
+- [ ] Type the next `cherry-picker-vX.Y.Z` version and select "**Create new tag: cherry-picker-vX.Y.Z** on publish"
+
+- [ ] Leave the "Release title" blank (it will be autofilled)
+
+- [ ] Click "Generate release notes" and amend as required
+
+- [ ] Click "Publish release"
```

Comment:
```suggestion
- [ ] Go to [Releases page](https://github.com/python/cherry-picker/releases) and

  - [ ] Click "Draft a new release"

  - [ ] Click "Choose a tag"

  - [ ] Type the next `cherry-picker-vX.Y.Z` version and select "**Create new tag: cherry-picker-vX.Y.Z** on publish"

  - [ ] Leave the "Release title" blank (it will be autofilled)

  - [ ] Click "Generate release notes" and amend as required

  - [ ] Click "Publish release"
```

Would it make sense to indent these as they are all actions to be performed in the Releases page?

Reply:
Sure!